pkg/rtc/types/interfaces.go

```diff
@@ -87,6 +90,9 @@ type Participant interface {
 	RemoveSubscribedTrack(st SubscribedTrack)
 	SubscriberPC() *webrtc.PeerConnection
 
+	UpdateSubscriptionPermissions(permissions *livekit.UpdateSubscriptionPermissions, resolver func(participantSid string) Participant) error
+	SubscriptionPermissionUpdate(publisherSid string, trackSid string, allowed bool)
```

Comment:
two new methods, one changed method, one removed method.

Probably need to add one more if we do `RemoveSubscriber` through the publisher participant.